flink-tests/src/test/java/org/apache/flink/test/javaApiOperators/GroupCombineITCase.java

```diff
@@ -58,7 +58,7 @@
 */
 public class GroupCombineITCase extends MultipleProgramsTestBase {
 
-	public GroupCombineITCase(ExecutionMode mode) {
+	public GroupCombineITCase(TestExecutionMode mode) {
```

Comment:
great! Thanks for the note. I will rebase once again from master and remove my quickfix